api/client/volume.go

```diff
@@ -286,13 +286,13 @@ func (cli *DockerCli) initVolumes(vols []string, reload bool) error {
 	return err
 }
 
-func uploadLocalVolume(cli *DockerCli, source, url, cookie string, results []error, wg *sync.WaitGroup) {
+func uploadLocalVolume(cli *DockerCli, source, url, cookie string, results *[]error, wg *sync.WaitGroup) {
 	var err error
 
-	fmt.Fprintf(cli.out, "uploading %s", source)
+	fmt.Fprintf(cli.out, "uploading %s ", source)
```

Comment:
That was intentional, so that the there is a space between source and upcoming error message.